### PR TITLE
Apply the rel=lightbox migration to all rte fields

### PR DESCRIPTION
**Pull request by ausi**
As discussed in https://github.com/contao/contao/issues/4505#issuecomment-1118608425

### CODE REVIEW

File: core-bundle/src/Migration/Version413/RelLightboxMigration.php [hunk 1]
```diff
@@ -86,6 +80,25 @@ public function run(): MigrationResult
...
+
+        foreach ($schemaManager->listTableNames() as $tableName) {
+            try {
+                Controller::loadDataContainer($tableName);
+            } catch (\Throwable $e) {
+                continue;
+            }
+
+            foreach ($GLOBALS['TL_DCA'][$tableName]['fields'] ?? [] as $fieldName => $fieldConfig) {
+                if ('tiny' === substr($fieldConfig['eval']['rte'] ?? '', 0, 4)) {
```
Comment (aschempp) [inline core-bundle/src/Migration/Version413/RelLightboxMigration.php:96]: should probably use `str_starts_with`?

**Review by aschempp — CHANGES_REQUESTED** [review]

File: core-bundle/src/Migration/Version413/RelLightboxMigration.php (hunk 1 above)
Comment (ausi) [reply core-bundle/src/Migration/Version413/RelLightboxMigration.php:96]: We’d have to require `symfony/polyfill-php80` then, don’t we?

**Review by leofeyer — APPROVED** [review]

File: core-bundle/src/Migration/Version413/RelLightboxMigration.php (hunk 1 above)
Comment (leofeyer) [reply core-bundle/src/Migration/Version413/RelLightboxMigration.php:96]: Yes, we could do that. But let‘s try not to use these features until we can use them natively. The current solution is working and Rector will automatically change it to `str_starts_with` when merging upstream.

**Comment by leofeyer** [comment]
Thank you @ausi.

**Comment by leofeyer** [comment]
Thank you @ausi.

**Pull request merged**